packages/core/src/actor.ts

```
@@ -1,142 +1,129 @@
-export type Actor = z.infer<typeof Actor>;
+  export interface Token {
+    type: "steam";
+    properties: {
+      steamID: bigint;
+    };
+  }
 
-export const ActorContext = createContext<Actor>("actor");
+  export interface Public {
+    type: "public";
+    properties: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)